packages/vm/src/vm.ts

```diff
@@ -78,107 +77,57 @@ export class VM {
    * @param opts VM engine constructor options
    */
   static async create(opts: VMOpts = {}): Promise<VM> {
-    if (opts.bn128 === undefined) opts.bn128 = await initRustBN()
-    const vm = new this(opts)
-
-    const genesisStateOpts =
-      opts.stateManager === undefined && opts.genesisState === undefined
-        ? { genesisState: {} }
-        : undefined
-    await vm.init({ ...genesisStateOpts, ...opts })
-    return vm
-  }
-
-  /**
-   * Instantiates a new {@link VM} Object.
-   *
-   * @deprecated The direct usage of this constructor is discouraged since
-   * non-finalized async initialization might lead to side effects. Please
-   * use the async {@link VM.create} constructor instead (same API).
-   * @param opts
-   */
-  protected constructor(opts: VMOpts = {}) {
-    this.events = new AsyncEventEmitter<VMEvents>()
-
-    this._opts = opts
-
-    if (opts.common) {
-      this.common = opts.common
-    } else {
+    // Add common, SM, blockchain, EVM here
+    if (opts.common === undefined) {
       const DEFAULT_CHAIN = Chain.Mainnet
```

Comment:
Do we really need this?  It feels unnecessary to declare this constant instead of just passing `Chain.Mainnet` in the next line to the `common` constructor.